SlevomatCodingStandard/Sniffs/TypeHints/DeclareStrictTypesSniff.php

```diff
@@ -234,4 +235,12 @@ public function process(File $phpcsFile, $openTagPointer): void
 		$phpcsFile->fixer->endChangeset();
 	}
 
+	protected function getStrictTypeDeclaration(): string
```

Comment:
Please fix CS.

Reply:
@wietsewarendorff And now please squash it to one commit with the commit message like others: https://github.com/slevomat/coding-standard/commits/master

Thank you.

Reply:
no problem, there it is